core/ledger/kvledger/txmgmt/statedb/statecouchdb/cache_test.go

```diff
@@ -57,6 +58,29 @@ func TestGetPutState(t *testing.T) {
 	require.True(t, proto.Equal(expectedValue1, v))
 }
 
+func TestGetPutStateWithBigPayloadIfKeyDoesNotExist(t *testing.T) {
+	cache := newCache(32, sysNamespaces)
+
+	// test GetState with empty cache
+	v, err := cache.getState("ch1", "ns1", "k1")
+	require.NoError(t, err)
+	require.Nil(t, v)
+
+	expectedValue := &CacheValue{Value: []byte("value")}
+	require.NoError(t, cache.putState("ch1", "ns1", "k1", expectedValue))
+
+	// test PutState with BigPayload
+	token := make([]byte, (64*1024)+1)
+	rand.Read(token)
+	expectedValue1 := &CacheValue{Value: token}
+	require.NoError(t, cache.putState("ch1", "ns1", "k1", expectedValue1))
+
```

Comment:
Similar to this, you could have added a test for the function `cache.UpdateStates`, as that is not covered in this test.